geary-core/src/main/kotlin/com/mineinabyss/geary/ecs/engine/Archetype.kt

```diff
@@ -27,11 +28,21 @@ public data class Archetype(
     // Currently all relations must hold data and the HOLDS_DATA bit on them corresponds to the component part.
     private val dataHoldingType = type.filter { it.holdsData() || it.isRelation() }
 
+    /**
```

Comment:
Personally I prefer single line commends for properties, or `@property` in the doc for the class itself